### DISCUSSION TITLE
Osu! not submitting scores on my steam deck

### DISCUSSION THREAD

**Discussion by SltySptoon** [ppy/osu#27170] [Q&A]
Hello, I downloaded osu lazer from the official osu website, ran the app image as an executable on steam os, logged in, set a score (linked in the image), yet my score does not submit, appear on leaderboards, or show up in my recently played maps. My profile and relevant images are linked below, thank you for your time. https://osu.ppy.sh/users/18234924
![IMG_0242](https://github.com/ppy/osu/assets/60854876/0b62bd98-2405-4609-884f-b26745af5b05)
![IMG_0241](https://github.com/ppy/osu/assets/60854876/39447a3e-8db4-4cb8-92ea-461f0551083e)

**Comment by bdach** [comment]
please provide logs. i don't know how you do it from a steam deck. other platforms are as in https://github.com/ppy/osu/discussions/15188.

**Reply by SltySptoon** [reply]
![IMG_0247](https://github.com/ppy/osu/assets/60854876/a1421709-8cbe-49d6-9421-ee5afa776e26)

when I tried the command I got this response. I am assuming that is not the actual logs so I will try more once I get back from school

**Reply by bdach** [reply]
you wanna attach the files from `~/.local/share/osu/logs` here probably.
